param/low_level_motion_sim_1.5.yaml

```diff
@@ -70,10 +70,12 @@ takeoff_throttle_ramp_duration: 0.5
 # Rate at which to descent
 # must be negative
 descend_rate: -0.7
+cushion_rate: -.05
```

Comment:
Can these be set to the same values as the crazyflie was tested with?

Reply:
Also these parameters need to be added to the sim (not 1.5) and 2.0

Reply:
Can you add a quick comment to explain the usage of these parameters? It will be helpful to us when we forget what they mean the night before competition.